### PR TITLE
feat: categories added

### PR DESCRIPTION
**Pull request by sharmayatendra**
## Changes 

- categories for quiz added
image

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -2,12 +2,13 @@ import "./App.css";
...
 import { Homepage } from "./pages/Homepage/Homepage";
+import { Categories } from "./pages/Categories/Categories";
 
 function App() {
   return (
     <Routes>
       <Route path="/" element={<Homepage />}></Route>
-
+      <Route path="/categories" element={<Categories />}></Route>
       {/* <Mockman colorScheme="dark" /> */}
```
Comment (tushargoyal812) [inline src/App.js:12]: remove comments if not required

**Review by tushargoyal812 — COMMENTED** [review]
looks good to merge

**Review by yashpatel1797 — APPROVED** [review]
code looks clean
file structure is good
design looks consistent

File: src/App.js (hunk 1 above)
Comment (sharmayatendra) [reply src/App.js:12]: ok

**Pull request merged**